server/db/db.js

```diff
@@ -668,9 +743,9 @@ async function clearButtons(pgClient) {
   try {
     await helpers.runQuery(
       'clearButtons',
-      `DELETE FROM buttons
+      `DELETE FROM devices WHERE device_type = $1
```

Comment:
I think this one can also be generalized to `clearDevices` and have it without the `WHERE` clause. It's only used  as part of `clearTables` in our integration tests, and it's intended to clear out the tables so that the tests start with a blank slate.

Reply:
Smart! I'll do this.